google-cloud-pubsub/lib/google/cloud/pubsub.rb

```diff
@@ -59,6 +59,8 @@ module PubSub
       # @param [Integer] timeout Default timeout to use in requests. Optional.
       # @param [Hash] client_config A hash of values to override the default
       #   behavior of the API client. Optional.
+      # @param [String] endpoint Override of the endpoint host name. Optional.
+      #   If the param is nil, uses the default endpoint.
```

Comment:
Can we document and/or provide an example value the expected format? (Specifically, that port is included: `localhost:4567`)

Reply:
@dazuma Do you want to show an example in the `endpoint` docs?

Reply:
I'd rather just document host name for now. (Yes technically it supports port as well, but I don't expect that to be a common use case initially, and there's a small chance we'll make a different interface decision across languages that Ruby might want to follow.)

Reply:
ok

Reply:
Right, we are being vague and only describing it as host name. But do we want to give an example of a host name value in the documentation?